packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -123,17 +123,15 @@ export type captionsOptions = {
 };
 
 // @internal
-export type _ChangeCaptionLanguageButtonSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
-    supportedCaptionLanguages: string[];
-    currentCaptionLanguage: string;
-};
-
-// @internal
-export type _ChangeSpokenLanguageButtonSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
+export type _ChangeSpokenLanguageSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
     supportedSpokenLanguages: string[];
     currentSpokenLanguage: string;
+    isCaptionsFeatureActive: boolean;
 };
 
+// @internal
+export const _changeSpokenLanguageSelector: _ChangeSpokenLanguageSelector;
```

Comment:
Is this a selector for the component? If yes, then name it as
[componentName]Selector

Reply:
It's a selector for multiple components, all having same purpose : change spoken language